### ISSUE TITLE
Could not import gateway api yaml

### ISSUE THREAD

**Issue by blmhemu** [grafana/tanka#1121]
Getting an error when trying to import the [gateway api yaml file](https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml).

Here is the code

```jsonnet
local gatewayapi = import './gatewayapi.yaml';

{
  gatewayapi: gatewayapi,
}

``` 

Error
```log
gatewayapi.yaml:48:36 Could not lex the character '?'
```

Relevant documentation
```
When using Tanka, it is also possible to directly import .json and .yaml files, as if they were a .libsonnet.
```

Also tried using

```jsonnet
local myYaml = std.parseYaml(importstr 'gateway.yaml');
```

Error snippet

```log
Error: got an error while extracting env `environments/gateway`: found invalid Kubernetes object (at .gateway): missing attribute "apiVersion"

gateway:
    - null
    - apiVersion: apiextensions.k8s.io/v1
      kind: CustomResourceDefinition
      metadata:
        annotations:
            api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/2997
            gateway.networking.k8s.io/bundle-version: v1.1.0
            gateway.networking.k8s.io/channel: standard
        creationTimestamp: null
        name: gatewayclasses.gateway.networking.k8s.io
      spec:
        group: gateway.networking.k8s.io
        names:
            categories:
                - gateway-api
            kind: GatewayClass
            listKind: GatewayClassList
            plural: gatewayclasses
            shortNames:
                - gc
            singular: gatewayclass
        scope: Cluster
```

The following snippet seems to work
```jsonnet
std.filter(
    function(x) x != null,
    gatewayYaml
  ),
```

**Comment by zerok** [comment]
Hi 🙂  I'm afraid that line in the docs seems to be wrong at this time, sorry about that. JSONNET itself does not support importing from YAML files directly and while Tanka adapts the import logic, I haven't found any YAML-related changes there. You will need to use `importstr` and `std.parseYaml`.

As for a simple `parseYaml` fails: The problem is that the first document in that file only consists of comments and so it does not represent a Kubernetes manifest. So your last approach is IMO the correct one 🙂 

I wrote above that the line seems to be wrong "at this time". I'm still checking if this might be a regression as I've found some fragments in the code that this might have been supported in the path. I'll update this issue once I know more 🙂
